coderd/notifications/notifications_test.go

```diff
@@ -764,7 +764,9 @@ func TestNotificationTemplatesCanRender(t *testing.T) {
 			payload: types.MessagePayload{
 				UserName: "Bobby",
 				Labels: map[string]string{
-					"created_account_name": "bobby",
+					"created_account_name":      "bobby",
+					"created_account_user_name": "William Tables",
+					"account_creator":           "rob",
```

Comment:
Why don't we use initiator for these fields like in other places. I feel like templates would be simplified if we didn't add the specific prefix either (created/deleted/etc.).

Reply:
fixed in subsequent PR